bower.json

```diff
@@ -1,20 +1,35 @@
 {
   "name": "html-imports",
   "main": "html-imports.min.js",
+  "description": "HTML Imports polyfill",
   "version": "1.0.0-rc.2",
-  "homepage": "http://webcomponents.org",
+  "homepage": "https://webcomponents.org/polyfills",
   "authors": [
-    "The Polymer Authors"
+    "The Polymer Project Authors (https://polymer.github.io/AUTHORS.txt)"
```

Comment:
Thanks for the update. Next time feel free to update `package.json` in this branch as well, since the change you made is the same in 2 different places 👌